Makefile

```diff
@@ -90,6 +90,9 @@ ip: ansible.cfg
 python: ansible.cfg
 	@ansible all -m raw -a 'sudo apt-get install --assume-yes python python-apt'
 
+etc-hosts: ansible.cfg
```

Comment:
Should this be part of `all`?

Reply:
I waffled on that.

I also waffled on naming it `.etc-hosts.yml` or otherwise hiding the playbook to keep the directory as clean as possible.

In the end - all the Makefile itself needs is itself, and a valid Vagrantfile (think `vagrant init`). An internet connection is not even required. Making etc-hosts part of all requires another file, or the ability (not yet implemented) to download it (à la `Vagrantfile-force`).

Either way, I do need to update the `make menu` target, add the `etc-hosts.yml` role to download, use tags to download (instead of hard coding master), and tag the commit.